### PR TITLE
logging improvements

### PR DESCRIPTION
**Pull request by gaorlov**
- adding tagging to logging to include the consumer class
- making zookeeper excptions log as 'error' rather than 'info'

### CODE REVIEW

File: lib/simple_kafka_consumer/consumer.rb [hunk 1]
```diff
@@ -34,12 +34,12 @@ def parse(message)
 
     def log(message)
       return false unless logger
-      logger.info message
+      logger.tagged(self.class) { logger.error message }
```
Comment (chingor13) [inline lib/simple_kafka_consumer/consumer.rb:37]: should probably be `info` to preserve backwards compatibility

Comment (chingor13) [inline lib/simple_kafka_consumer/consumer.rb:37]: oh, that's a feature

**Comment by chingor13** [comment]
I don't see why this is needed. It's adding a dependency on the tagged logger. Why not pass in a tagged logger when you create the consumer?

**Comment by gaorlov** [comment]
do you mean like

``` ruby
def initialize(*args)
  @logger = ActiveSupport::TaggedLogging.new(Logger.new(STDOUT)).tagged( self.class.to_s )
end
```

because you can't do that : (

```
2.3.1 :001 > logger = ActiveSupport::TaggedLogging.new(Logger.new(STDOUT))
 => #<Logger:0x000000017d0a40 > 
2.3.1 :002 > a = logger.tagged( "lol" )
LocalJumpError: no block given (yield)
```

**Pull request closed without merging**